src/components/molecules/DateInput/DateInput.tsx

```diff
@@ -0,0 +1,87 @@
+import { ChangeEvent, FC, useEffect, useState } from 'react'
+import { isValid, parse } from 'date-fns'
+import { Input, InputGroup, InputRightElement, Text, VStack } from '@chakra-ui/react'
+import { CalendarIcon } from '@/components/atoms/CalendarIcon/CalendarIcon'
+
+interface DateInputProp {
+  label: string
+  openDatePicker: () => void
```

Comment:
`onOpenDatePicker`. Generally, all event handlers passed as props should be named starting with `on`